frontend/ui/quickstart.lua

```diff
@@ -63,16 +84,88 @@ local quickstart_guide = T(_([[
 ---
 You can access the complete user manual from [our GitHub page](https://github.com/koreader/koreader).
 ]])
-..
-_([[## User interface <a id="ui"></a>
+end
+-- User interface
+if Device:hasScreenKB() then
+    quickstart_guide = quickstart_guide .. _([[## User interface <a id="ui"></a>
+
+<div class="img-block">![Touch zones](resources/quickstart/kindle4.png)</div>
+
+- To show the **TOP MENU** or **BOTTOM MENU** press the **Menu** or **Press** keys respectively.
+- The **STATUS BAR** can be set to show a multitude of information regarding your reading progress or device state.
+]])
+elseif Device:hasSymKey() then
+    quickstart_guide = quickstart_guide .. _([[## User interface <a id="ui"></a>
+
+<div class="img-block">![Touch zones](resources/quickstart/kindle3.png)</div>
+
+- To show the **TOP MENU** or **BOTTOM MENU** press the **Menu** or **Aa** keys respectively.
+- The **STATUS BAR** can be set to show a multitude of information regarding your reading progress or device state.
+]])
+else
+    quickstart_guide = quickstart_guide .. _([[## User interface <a id="ui"></a>
 
 <div class="img-block">![Touch zones](resources/quickstart/touchzones.png)</div>
 
 - To show the **TOP MENU** or **BOTTOM MENU** you can click the indicated zones. You can click or swipe down the upper zone to show the **TOP MENU**.
 - The **STATUS BAR** zone can be used to cycle between STATUS BAR items if one item is visible. This will also hide and show the STATUS BAR if you tap enough times.
 ]])
-..
-_([[## User interface tips <a id="uitips"></a>
+end
+-- User interface tips
```

Comment:
And you can add a blank line to make it easier to read.
(We didn't with `..` as it would look odd.)